api/share/requests/users.py

```diff
@@ -0,0 +1,65 @@
+import re
+from api.share.requests import ValidRequestObject, InvalidRequestObject
+
+
+class GetUsersCountRequest(ValidRequestObject):
+
+    def __init__(self,
+                 registration_bot_chat_id: str,
+                 tg_username: str,
+                 phone: str,
+                 email: str,
+                 first_name: str,
+                 last_name: str):
+        self.registration_bot_chat_id = registration_bot_chat_id
+        self.tg_username = tg_username
+        self.phone = phone
+        self.email = email
+        self.first_name = first_name
+        self.last_name = last_name
+
+    @classmethod
+    def from_dict(cls, input_dict):
+        # Check input parameters.
+        invalid_request = InvalidRequestObject()
+        # https://core.telegram.org/bots/api#chat : chat_id - max 52 significant bits
+        # 9007199254740991 is the max number you can store in the 53 bits => len(chat_id) <= 16
+        if input_dict.get("registration_bot_chat_id") and \
+                len(input_dict.get("registration_bot_chat_id")) > 16:
+            invalid_request.add_error("registration_bot_chat_id", "Invalid length")
+
+        if input_dict.get("tg_username") and (len(input_dict.get("tg_username")) < 5 or
+                                              len(input_dict.get("tg_username")) > 32):
```

Comment:
same. I don't think `black` would like this :) I suggest we use it to keep our coding style consistent.

Same remark re: constants. It's best to keep "magic numbers" out of the code.

Reply:
> same. I don't think `black` would like this :) I suggest we use it to keep our coding style consistent.
> 
> Same remark re: constants. It's best to keep "magic numbers" out of the code.

Do we suppose these numbers will change? If no and if we don't use them somewhere else and if we have comments about these values, what is the benefit to keep them in constants?

Reply:
Well I don't know, it's just [good coding](https://refactoring.guru/replace-magic-number-with-symbolic-constant) style to me, especially when same number is repeated over and over again. I can understand magic numbers in tests, but not in working code, where they are repeated.

Reply:
Ok, I got it